underfs/tutorial/README.md

```diff
@@ -8,19 +8,18 @@ Dummy Alluxio under storage implementation accessing local file system using the
 mvn package -DskipTests
 ```
 
-### Run Under Storage Contract Tests
-
-Note: The following runs a test to verify that the under storage satisfies the contract with Alluxio.
-Without the option `testTutorialUFS` only unit tests are triggered.
+### Deploy
 
 ```bash
-mvn test -DtestTutorialUFS=dummy://<folder>
+/bin/alluxio extensions install <path>/<to>/alluxio-extensions/underfs/tutorial/target/alluxio-underfs-dummy-1.6.0-SNAPSHOT.jar
```

Comment:
nit: not necessarily because of this PR:
- add `.` before `/bin`
- replace 1.6.0 with `{{site.ALLUXIO_RELEASED_VERSION}}` throughout this doc.